submarine/src/main/java/org/apache/zeppelin/submarine/IPySubmarineInterpreter.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.zeppelin.submarine;
+
+import org.apache.zeppelin.interpreter.InterpreterException;
+import org.apache.zeppelin.python.IPythonInterpreter;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Properties;
+
+public class IPySubmarineInterpreter extends IPythonInterpreter {
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(IPySubmarineInterpreter.class);
+
+  private PySubmarineInterpreter pySubmarineInterpreter;
+
+  public IPySubmarineInterpreter(Properties properties) {
+    super(properties);
+  }
+
+  @Override
+  public void open() throws InterpreterException {
+    PySubmarineInterpreter pySparkInterpreter =
+        getInterpreterInTheSameSessionByClassName(PySubmarineInterpreter.class, false);
+
+    pySubmarineInterpreter
```

Comment:
`pySubmarineInterpreter` is never used ?

Reply:
Currently we are using the `pySubmarineInterpreter`
Later, when we supported the `IpySubmarineInterpreter`, I will add it again.
I refer to the code of the `Zeppelin Python Interpreter`,
I am not familiar with `ipython` at present, the current code, It can be used normally,
Therefore, in order to stabilize the system, still retain this part of the code.